packages/paste-website/src/pages/components/status-badge/index.mdx

```diff
@@ -97,8 +98,16 @@ export const getStaticProps = async () => {
 
 ## About Status Badge
 
+The Status badge is an implementation of the the [Status Pattern](/patterns/status), pre-built for you to display and indicate the status of a process or connection in your product.
+
+It pairs Status level with color and icon to assist with visual design and accessbility.
```

Comment:
```suggestion
Status badge is an implementation of the [Status Pattern](/patterns/status) to display the status of a process or connectivity in your product.
```

Reply:
Removed the last line since it's repeated in the Accessibility section right below.